bench/run_bench.py

```
@@ -0,0 +1,576 @@
+    return None, None, time.time() - t0, {}
+
+
+def pmc_run(pmc, cmd):
+    """Run `cmd` inside a pmc_tool counter window; return (events, elapsed).
+
+    pmc_tool deliberately does not run `cmd` itself — it is setuid-root, and a
+    setuid binary that execs a caller-supplied command is a local privilege
+    escalation. It opens a counter window, prints READY, and waits on stdin; we
+    run `cmd` ourselves (unprivileged) in the gap and then close the window.
+    """
+    proc = subprocess.Popen(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)